server/testserver.go

```diff
@@ -15,24 +15,24 @@
 //
 // Author: Spencer Kimball (spencer.kimball@gmail.com)
 
+// A TestServer encapsulates an in-memory instantiation of a cockroach
+// node with a single store. Example usage of a TestServer follows:
+//
+//   s := server.StartTestServer(t)
+//   defer s.Stop()
+//
+
 package server
 
 import (
 	"testing"
 
-	"github.com/cockroachdb/cockroach/gossip"
-	"github.com/cockroachdb/cockroach/proto"
-	"github.com/cockroachdb/cockroach/security"
-	"github.com/cockroachdb/cockroach/storage"
-	"github.com/cockroachdb/cockroach/storage/engine"
-	"github.com/cockroachdb/cockroach/util"
-	"github.com/cockroachdb/cockroach/util/hlc"
 	"github.com/cockroachdb/cockroach/util/log"
 )
 
 // StartTestServer starts a in-memory test server.
-func StartTestServer(t testing.TB) *TestServer {
-	s := &TestServer{}
+func StartTestServer(t testing.TB) *EmbeddedServer {
```

Comment:
You can remove the dependency on `testing` here but changing `StartTestServer` to take a tiny interface containing a `Fatalf` method:

```
  type tester interface {
    Fatalf(format string, args ...interface{})
  }
```

Or you could have it take a function with the signature of `Fatalf`:

```
func StartTestServer(fatalf func(format string, args ...interface{})) {
}
```